vendor/processors/__init__.py

```diff
@@ -1,4 +1,6 @@
 from vendor.config import VENDOR_PAYMENT_PROCESSOR
 from django.utils.module_loading import import_string
 
-PaymentProcessor = import_string('vendor.processors.{}'.format(VENDOR_PAYMENT_PROCESSOR))
\ No newline at end of file
+def get_payment_processor():
```

Comment:
Did this break?  Normally you want to do it the other way so it's immutable.

Reply:
In trainingcamp project. In the core/models/user.py of I try to import the PaymenteProcessor (`form vendor.processor import PaymentProcessor`) I got an error that the file were not ready to import as what was happening with the Product Model. 
That is why I created a wrapper, so I could import the PaymentProcessor and added to the signal on_user_signup to attach a student license to schools who are invite only.